docs/kp_secret_create.md

```diff
@@ -1,13 +1,15 @@
 ## kp secret create
 
-Create a secret configuration
+Create a secret for a service account
```

Comment:
```suggestion
Create a secret and link it to a service account
```
?

Reply:
How about `Create a secret and add it to a service account`?

Reply:
I like that.